README.md

```diff
@@ -7,68 +7,132 @@ PADD (formerly Chronometer2) is a more expansive version of the original chronom
 ***Note:** PADD has been adopted by the Pi-hole team, thanks to JPMCK for creating this helpful tool!
 
 ## Setup PADD
-*More in-depth information about setting up PADD can be found in this repo’s [wiki](https://github.com/pi-hole/PADD/wiki/Setup).*
 
 - Get a copy of PADD by running:
-
-```bash
-cd ~
-wget -O padd.sh https://install.padd.sh
-```
-or
-```bash
-cd ~
-curl -sSL https://install.padd.sh -o padd.sh
-```
+  ```bash
+  cd ~
+  wget -O padd.sh https://install.padd.sh
+  ```
+  or
+  ```bash
+  cd ~
+  curl -sSL https://install.padd.sh -o padd.sh
+  ```
 
 - Make PADD executable by running
+  ```bash
+  sudo chmod +x padd.sh
+  ```
 
-```bash
-sudo chmod +x padd.sh
-```
+## Using PADD
 
-- Set PADD to auto run by adding the following to the end of `~/.bashrc`:
+### PADD on Pi-hole machine
+<!-- See https://github.com/pi-hole/FTL/pull/1999 for why i worded it this way -->
+- if running as user belonging to the group `pihole` (if not use `sudo`)
```

Comment:
I see why you worded it like this, but I disagree: even when the user is not member of group `pihole` you don't need `sudo` to run PADD. Having access to `/etc/pihole/cli_password` is not required, it only makes it a litte bit more comfortable. If you don't access it, you will be ask to enter the password (and maybe the 2FA token) but no root powers are necessary.

Reply:
Gotcha 

will fix